content/en/apps/tutorials/local-setup.md

```diff
@@ -10,7 +10,7 @@ relatedContent: >
 ---
 
 {{% pageinfo %}}
-This tutorial will take you through setting up a local environment to build and test CHT applications. This includes setting up the necessary tools to download and run the CHT public docker image as well as a command line interface tool to manage and build CHT apps.
+This tutorial will take you through setting up a local environment to build and test CHT applications on CHT version 3.9.1. This includes setting up the necessary tools to download and run the CHT public docker image as well as a command line interface tool to manage and build CHT apps.
```

Comment:
Wish we had a way to make this dynamic on the version number.  It's a manual maintenance item and in docs sites that usually means it gets forgotten.

Reply:
There's [custom variables in hugo](https://www.mikedane.com/static-site-generators/hugo/variables/) which we could try and leverage. However, maybe when ever we update the docker instance we have a mini release process that updates this text?  Currently it looks to be only in one place.  Alternately, we could say something along the lines of "See our [Supported Versions](https://docs.communityhealthtoolkit.org/core/overview/supported-software/) page" and then denote which version the docker image is on and _this_ to the docker image release process?  That way we'd only have to reference the one `supported-software` page.

 Also, maybe we solve it when we solve when [we unify our install docs](https://github.com/medic/cht-docs/issues/317)?

Reply:
works for me.